corehq/apps/users/forms.py

```diff
@@ -1269,6 +1268,12 @@ def clean_search_string(self):
             raise forms.ValidationError(_("* and ? are not allowed"))
         return search_string
 
+    def clean_domains(self):
+        domains = self.data.getlist('domains[]', [self.domain])
+        if self.domain not in domains:  # always include the current domain
```

Comment:
Was this an explicit requirement? I can imagine a user wanting to download only a particular set of counties and not include the state-level domain.